easybuild/easyconfigs/j/jbigkit/jbigkit-2.1_fix-libpath.patch

```diff
@@ -0,0 +1,26 @@
+build process does not find its own libraries
+
+diff -ruN jbigkit-2.1.orig/Makefile jbigkit-2.1/Makefile
+--- jbigkit-2.1.orig/Makefile	2014-03-27 11:47:15.000000000 -0700
++++ jbigkit-2.1/Makefile	2019-03-21 12:05:55.442478000 -0700
+@@ -4,7 +4,7 @@
+ CC = gcc
+ 
+ # Options for the compiler: A high optimization level is suggested
+-CFLAGS = -O2 -W -Wno-unused-result
++CFLAGS = -O2 -W -Wno-unused-result -I../libjbig
```

Comment:
Preferably it should pick up the CFLAGS settings from EasyBuild, i.e. CFLAGS += -W -Wno-unused...
and not set -O2